web/src/components/PoiListItem.tsx

```diff
@@ -1,65 +1,42 @@
+import Avatar from '@mui/material/Avatar'
 import Divider from '@mui/material/Divider'
+import ListItem from '@mui/material/ListItem'
+import ListItemAvatar from '@mui/material/ListItemAvatar'
+import ListItemButton from '@mui/material/ListItemButton'
+import ListItemText from '@mui/material/ListItemText'
+import Typography, { TypographyProps } from '@mui/material/Typography'
 import { styled } from '@mui/material/styles'
 import React, { ReactElement } from 'react'
 import { useTranslation } from 'react-i18next'
 
 import { PoiModel } from 'shared/api'
 
 import { PoiThumbnailPlaceholder } from '../assets'
-import { helpers } from '../constants/theme'
-import Button from './base/Button'
+import Link from './base/Link'
 
-const ListItemContainer = styled('ul')`
-  font-family: ${props => props.theme.legacy.fonts.web.contentFont};
-  display: flex;
-  padding: clamp(10px, 1vh, 20px) 0;
-  cursor: pointer;
+const StyledListItemButton = styled(ListItemButton)`
+  align-items: flex-start;
+  gap: ${props => props.theme.spacing(2)};
+` as typeof ListItemButton
```

Comment:
Add Padding 12 Top-Bottom and 16 for Left-Right
  `min-height` to keep the visual weight consistent 🙂.
  ```js
  padding: 12px 16px;
  min-height: 80px;
  ```

Reply:
<img width="396" height="232" alt="image" src="https://github.com/user-attachments/assets/2ccceda9-d51c-4e32-8165-e0633f0e7e5c" />